spinalcordtoolbox/gui/sagittal.py

```diff
@@ -92,7 +92,7 @@ def launch_sagittal_dialog(input_file, output_file, params):
     if not params.vertebraes:
         params.vertebraes = [3, 5]
     params.input_file_name = input_file.absolutepath
-    params.subtitle = u'Use Left/Right arrows to navigate the right-left direction'
+    params.subtitle += u'\nUse Left/Right arrows to navigate the right-left direction'
```

Comment:
Trouble if `launch_sagittal_dialog()` is called several times... is that possible?

Reply:
I don't think it is. If it is we will fix it in another way, but for now i'd like to merge this PR before  https://github.com/neuropoly/spinalcordtoolbox/pull/1684 and https://github.com/neuropoly/spinalcordtoolbox/pull/1661 are merged (because they conflict)